packages/git/src/browser/history/git-history-contribution.ts

```diff
@@ -43,6 +46,27 @@ export class GitHistoryContribution extends AbstractViewContribution<GitHistoryW
         });
     }
 
+    @postConstruct()
+    protected init() {
+        this.repositoryTracker.onDidChangeRepository(async repository => {
+            if (repository) {
```

Comment:
What about the case when `repository` is `undefined`?

Reply:
You are right @akosyakov . I also changed git-history-widget.ts to handle this case.